src/App.tsx

```diff
@@ -1,6 +1,38 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import './App.scss';
+import classNames from 'classnames';
+
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
+export function getReorderedGoods(
+  goods: string[], sortType: number, isReversed: boolean,
+) {
+  // To avoid the original array mutation
+  const visibleGoods = [...goods];
+
+  visibleGoods.sort((good1, good2): number => {
+    switch (sortType) {
+      case SortType.ALPHABET:
+        return good1.localeCompare(good2);
+        break;
+      case SortType.LENGTH:
+        return good1.length - good2.length;
+      default:
+        return 0;
```

Comment:
would be better to throw some error, or show some message to user

Reply:
fix it please
https://github.com/mate-academy/react_list-of-goods-hooks/pull/671/files#r1091812856